hooks/post_gen_project.py

```diff
@@ -201,7 +201,8 @@ def update_local_workflows(central_workflows):
 
 
 def main():
-    add_supermodules(ROOT, "{{ cookiecutter.project_name }}")
+    if "." in "{{ cookiecutter.project_name }}":
```

Comment:
"if we have "." within the project, then it's a namespaced package, so deviate from the regular cookiecutter style which andrew has written in `add_supermodules`"

Reply:
To note: `add_supermodules` also uses the existence of "." to recognize it as a namespace package.

```
    # Get the necessary modules given the period spacings
    module_names = name.split(".")
```